### PR TITLE
OpenXLA pin update to 670dd9472b

### PR DESCRIPTION
**Pull request by cota**

### CODE REVIEW

**Comment by yeounoh** [comment]
Hey @cota , can we close this since https://github.com/pytorch/xla/pull/6677 is merged now? cc @miladm @lsy323

File: setup.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
 
 base_dir = os.path.dirname(os.path.abspath(__file__))
 
-_libtpu_version = '0.1.dev20240213'
+_libtpu_version = '0.1.dev20240220'
```
[the remark this answers is not in the document]
Comment (miladm) [reply setup.py:67]: update: https://github.com/pytorch/xla/pull/6677

thanks @yeounoh for pointing out the latest merge.

**Comment by miladm** [comment]
@cota does this PR aim to land in 2.3? I see it's in draft mode.

**Comment by cota** [comment]
Closing this now as it was superseded by https://github.com/pytorch/xla/pull/6677.

**Pull request closed without merging**